schema/core/reusable/param_int.json

```diff
@@ -0,0 +1,22 @@
+{
+    "schemaId": "core-reusable-param-int",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "integer parameter schema",
+    "properties": {
+        "value": {
+            "description": "Value of parameters",
+            "type": "integer"
+        },
+        "units": {
+            "description": "Unit of parameter",
+            "type": "string"
+        },
```

Comment:
The two below can be inherited from `scalar`, no?

Reply:
I don't think we can use `scalar` in more places than just `param_number.json`. The field `units` and `precision` cannot be inherited from another parameter schema with `allOf`, but we could use definitions/references. I'll include a draft of this in the next version.